File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/dip/excercise_00/Fan.java
package es.emergya.labs.training.solid_java_1.dip.excercise_00;

public class Fan implements Device{
	
	private int color;

	public void turnOn() {

		System.out.println("Fan turned on");
	}

	public void turnOff() {

		System.out.println("Fan turned off");
	}

	public int getColor() {

		return color;
	}

	public void setColor(int color) {

		this.color = color;
	}
}

File: labs-training-master/SolidPythonResuelto/main/dip/Fan.py
from Device import Device

class Fan(Device):
    def turnOn(self) -> None:
        print("Fan turned on")
    def turnOff(self) -> None:
        print("Fan turned off")
    def  getColor(self) -> None:
        return  self.color
    def setColor(self, color:int) -> None:
        self.color:int = color